topics/xml/index.md

```diff
@@ -0,0 +1,11 @@
+---
+created_by: World Wide Web Consortium
+display_name: XML
+released: 1996
+short_description: XML is subset of SGML (standard general markup language) used to store and transport data.
```

Comment:
I think SGML should be expanded `Standard Generalized Markup Language`

Reply:
- `general` to `generalized`